index.html

```diff
@@ -31,9 +31,144 @@ <h1 class="introduction"> Hey there. I’m Chukwuemeka <br> <span> I’m a softw
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton" type="submit"> See Project</button>        
         
+                            </div>
+
+                </div>  
+                <div id="recentcard" class ="fourthbox">
+                        <div class="innerdownbox">
+                                <p> Multi-Post Stories<br> Gain+Glory </p>
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton" type="submit"> See Project</button>        
+        
+                            </div>
+                </div>
+                <div id="recentcard" class="fifthbox">   
+                        <div class="innerdownbox">
+                                <p> Multi-Post Stories<br> Gain+Glory </p>
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton" type="submit"> See Project</button>        
+        
+                            </div>
+
+                </div>
+                <div id="recentcard" class="sixthbox">
+                        <div class="innerdownbox">
+                                <p> Multi-Post Stories<br> Gain+Glory </p>
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton" type="submit"> See Project</button>        
+        
+                            </div>
+                </div>  
+            </div>
+            
+ <!-- About me Section            -->
+
+            <article id="aboutme">
+                    <h2> About me</h2>
```

Comment:
- [OPTIONAL] Kindly add the right-side background image as in the template provided. Will create a better UI. As not required by the previous reviewer I consider it optional.

Figma template
<img width="372" alt="Screen Shot 2022-12-01 at 11 29 06 PM" src="https://user-images.githubusercontent.com/47673311/205202836-4f89613f-0298-4049-ba92-6e53facac31c.png">

Your design
<img width="372" alt="Screen Shot 2022-12-01 at 11 27 45 PM" src="https://user-images.githubusercontent.com/47673311/205202887-9b836f33-2907-4b9c-a07a-bdcc5bade9cc.png">